client/state/jetpack-remote-install/reducer.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { createReducer, combineReducers, keyedReducer } from 'state/utils';
+import {
+	JETPACK_REMOTE_INSTALL,
+	JETPACK_REMOTE_INSTALL_FAILURE,
+	JETPACK_REMOTE_INSTALL_SUCCESS,
+} from 'state/action-types';
+
+export const isComplete = keyedReducer(
+	'url',
+	createReducer( false, {
+		[ JETPACK_REMOTE_INSTALL_SUCCESS ]: () => true,
+		[ JETPACK_REMOTE_INSTALL ]: () => false,
+	} )
+);
+
+export const inProgress = keyedReducer(
```

Comment:
Wait, this reducer will not be necessary, I just found out about data-layer `trackRequest` and the `getRequest()` selector for telling if a request is in-flight.

Reply:
Even better!